lektor/imagetools.py

```diff
@@ -17,6 +20,25 @@
 datetime.strptime('', '')
 
 
+class THUMBNAIL_MODE(IntEnum):
+    FIT, CROP, STRETCH = range(1, 4)
+
+    @property
+    def label(self):
+        return self.name.lower().replace('_', '-') # pylint: disable=no-member
+
+    @classmethod
+    def from_label(cls, label):
+        name = label.upper().replace('-', '-')
```

Comment:
Yeah, it's useless now since there are only "fit", "crop" and "stretch" modes. But other modes I was considering were "liquid-rescale" and "smart-crop", which would cause accessing `ThumbnailMode.LIQUID_RESCALE` and `ThumbnailMode.SMART_CROP`.

I'll add a bit of docs to that thing.